### PR TITLE
Add keycloak authorization capability

### PR DESCRIPTION
**Pull request by HerrDerb**

### CODE REVIEW

File: core/deployment/src/main/java/io/quarkus/deployment/Capability.java [hunk 1]
```diff
@@ -108,6 +108,8 @@ public interface Capability {
 
     String OIDC = QUARKUS_PREFIX + "oidc";
 
+    String KEYCLOAK_AUTHORIZATION = QUARKUS_PREFIX + ".keycloak.authorization";
```
Comment (sberyozkin) [inline core/deployment/src/main/java/io/quarkus/deployment/Capability.java:111]:
```suggestion
    String KEYCLOAK_AUTHORIZATION = QUARKUS_PREFIX + "keycloak.authorization";
```

**Review by sberyozkin — APPROVED** [review]

**Review by sberyozkin — APPROVED** [review]

**Comment by HerrDerb** [comment]
Can we merge this? Then I could fix my workaround 🤞

**Comment by sberyozkin** [comment]
@HerrDerb Sure, thanks

**Comment by HerrDerb** [comment]
@sberyozkin I thank you 🙂

**Pull request merged**